scripts/sync.js

```diff
@@ -21,6 +21,7 @@ program
   .option('--target_arch <target_arch>', 'target architecture')
   .option('--target_apk_base <target_apk_base>', 'target Android OS apk (classic, modern, mono)')
   .option('--submodule_sync', 'run submodule sync')
+  .option('--ignore_chromium', 'do not update chromium')
```

Comment:
I'm not sure about adding more flags here when we're trying to simplify things, I thought we were going to check for patches changes to decide if we needed to update or not?

Reply:
It would be great to be automatic but until then I think we need this. When in a crunch and needing to update to branches which have an incremental update, I often can't wait for a full build. But if you see more downsides, happy to hear. I agree we need to be careful with flags, but I'd say this one is ok since it doesn't drastically change functionality, and is easy to understand.